content/docs/components-and-props.md

```diff
@@ -42,27 +42,27 @@ class Welcome extends React.Component {
 }
 ```
 
-The above two components are equivalent from React's point of view.
+Os dois componentes acima são equivalentes do ponto de vista do React.
 
-Classes have some additional features that we will discuss in the [next sections](/docs/state-and-lifecycle.html). Until then, we will use function components for their conciseness.
+Classes tem alguns recursos adicionais que nós discutiremos nas [próximas seções](/docs/state-and-lifecycle.html). Até lá, nós usaremos componentes de função para sua concisão.
```

Comment:
```suggestion
Classes tem alguns recursos adicionais que nós discutiremos nas [próximas seções](/docs/state-and-lifecycle.html). Até lá, nós usaremos componentes de função por serem mais sucintos.
```
Albeit "concisão" is the exact translation of "conciseness", I think that's not so widespread in Portuguese and could worsen the understanding. I changed to a synonym phrase to keep the meaning but also improve understanding. 

Is that ok to you?

Reply:
Yes, that's better.